subscription-service/src/test/kotlin/com/egm/stellio/subscription/service/SubscriptionServiceTests.kt

```diff
@@ -899,4 +901,54 @@ class SubscriptionServiceTests : WithTimescaleContainer {
 
         assertEquals(res, true)
     }
+
+    /**
+     * Test for function 'getAllSubscriptions' with two subscription.
+     * One with the 'lastNotification' attribute set to null and the other with the 'lastNotification' now and
+     * 'timeInterval' set to 1
+     *
+     * @return list of subscription
+     */
+    @Test
+    fun `it should return all Entity where 'timeInterval' are expired `() {
+
+        val persistedSubscription = subscriptionService.getById(subscription1Id).block()!!
+        val notification = Notification(subscriptionId = subscription1Id, data = emptyList())
+
+        runBlocking {
+            subscriptionService.updateSubscriptionNotification(persistedSubscription, notification, true)
+                .then(subscriptionService.getById(subscription1Id)).awaitFirst()
```

Comment:
don't really get what you are trying to do here:
- do you really need to get the subscription by id?
- what is the purpose of updating the notification?

Reply:
1. I delete that `then(subscriptionService.getById(subscription1Id))` because I don't need 
2. I update the notification to update `lastNotification` attribute